src/utils/global/ambassadorCard/AmbassadorCard.tsx

```diff
@@ -73,7 +75,30 @@ export default function AmbassadorCard(props: AmbassadorCardProps) {
         </div>
 
         <div className={styles.row}>
-          <h3>Conservation Status</h3>
+          <div className={styles.col}>
+            <h3>Conservation Status</h3>
+          </div>
+          <div className={styles.col}>
+            <Tooltip
+              text="An objective assessment system for classifying the status of plants, animals, and other organisms threatened with extinction."
+              title="IUCN"
+              leftOffset="60%"
+              textContainerWidth="18rem"
+            >
```

Comment:
```suggestion
          <div className={styles.info}>
            <h3>Conservation Status</h3>
            <Tooltip
              text="An objective assessment system for classifying the status of plants, animals, and other organisms threatened with extinction."
              title="IUCN"
              leftOffset="60%"
              textContainerWidth="18rem"
              className={styles.tooltip}
            >
```
See related comment with the SCSS changes, and this also relies on my other suggestion about support a className in tooltip that is passed to the wrapper